.gitattributes

```diff
@@ -38,68 +38,81 @@
 #/Changes                 merge=union
 
 # No header for text files (would be too obtrusive).
-*.md                     ocaml-typo=missing-header
-README*                  ocaml-typo=missing-header
-*.adoc                   ocaml-typo=missing-header,long-line,unused-prop
-
-/.mailmap                ocaml-typo=long-line,missing-header,non-ascii
-/.merlin                 ocaml-typo=missing-header
-/Changes                 ocaml-typo=utf8,missing-header
-/INSTALL                 ocaml-typo=missing-header
-/LICENSE                 ocaml-typo=long-line,very-long-line,missing-header
+*.md                     typo.missing-header
+README*                  typo.missing-header
+*.adoc                   typo.missing-header
+*.adoc                   typo.long-line=may
+
+/.mailmap                typo.long-line typo.missing-header typo.non-ascii
+/.merlin                 typo.missing-header
+/Changes                 typo.utf8 typo.missing-header
+/INSTALL                 typo.missing-header
+/LICENSE                 typo.long-line typo.very-long-line typo.missing-header
```

Comment:
@dra27 I thought it was doing that already? Did I just dream it?

Reply:
No - you definitely need both; it could be done by having the test for `long-line` being between 81 and 131 characters and then `very-long-line` being > 132.

Reply:
Oops!

Reply:
This is a bit tricky (as in: it requires me to rediscover `awk` again). Given the large latency on my side, I won't touch this feature request and leave it for a future PR.